src/pivot-table/components/shared-styles.ts

```diff
@@ -74,18 +74,29 @@ export const stickyCell: Pick<React.CSSProperties, "width" | "maxWidth" | "posit
   top: 4,
 };
 
-export const getBorderStyle = (isLastRow: boolean, isLastColumn: boolean, borderColor: string) => {
-  if (isLastRow && isLastColumn) {
-    return cellStyle;
-  }
+export const getBorderStyle = (
```

Comment:
We are putting too much into this method. There are many different types of borders in the pivot table so this will only grow

Reply:
Yes, I agree. 
Probably we are going to need to revisit this for a refactor in future. but let's keep it away from this pr since if we try to fix it here it will only grow and keep this pr away from it's main purpose (last page empty rows). 

I'll create a refactor ticket in Jira for it.